v19.1/sql-statements.md

```diff
@@ -169,7 +169,7 @@ Statement | Usage
 
 ## Changefeed statements (Enterprise)
 
-[Change data capture](change-data-capture.html) (CDC) provides row-level change feeds into Apache Kafka for downstream processing.
+[Change data capture](change-data-capture.html) (CDC) provides row-level change feeds into [Apache Kafka](https://kafka.apache.org/) for downstream processing.
```

Comment:
This addition isn't necessary, as the primary link is to change data capture page in our docs where there's already a link to Kafka.